ftw/contacts/resources/contactfolder_listing.js

```diff
@@ -0,0 +1,81 @@
+$(function() {
+    ContactFolderListing.init();
```

Comment:
@bierik can this create load order problems in certain situations, as the init is called in a onload event as well as the definition? Or am I getting it wrong?